cypress/e2e/core_suite_master.cy.js

```diff
@@ -0,0 +1,674 @@
+    cy.get('a > .inline-flex').click()
+    cy.contains('Save Changes').click()
+  })
+
+  it.skip('3.1 edit homepage', () => {
+    _login()
+    cy.wait(500)
+    cy.contains('cf').click()
+    cy.contains('Dashboard').click()
+    cy.contains('Edit').click()
+    cy.contains('Edit Homepage').click()
+    cy.contains('Edit Banner and Logo').click()
+    cy.contains('Save Changes').click()
+    cy.wait(2000)
+    cy.scrollTo('top')
+    cy.get(':nth-child(1) > .p-2 >')
+      .trigger('dragstart', { force: true })
+
+      .invoke('css', 'color', 'red')
+    cy.wait(2000)
+    cy.get(':nth-child(2) > .p-2 >')
+      .trigger('drop', { force: true })
+      .invoke('css', 'color', 'blue')
+    //   .trigger('mouseover')
+    //   .trigger('mousedown', { which: 1, force: true })
+    //   .trigger('mousemove', {
+    //     clientX: 276,
+    //     clientY: 339,
+    //     screenX: 276,
+    //     screenY: 339,
+    //     pageX: 276,
+    //     pageY: 339,
+    //   })
+    //   .trigger('mouseup', { force: true, which: 1 })
+  })
+
+  it('4.0 - site nav - dictionary', () => {
+    cy.visit(`${Cypress.env('baseUrl')}${Cypress.env('DIALECT')}`)
+    cy.contains('Dictionary').click()
+    cy.contains('Words').click()
+    cy.contains('WORDS').should('exist')
+
+    cy.contains('Dictionary').click()
+    cy.contains('Phrases').click()
+    cy.contains('PHRASES').should('exist')
+
+    cy.contains('Dictionary').click()
+    cy.contains('Alphabet').click()
+    cy.contains('ALPHABET').should('exist')
+
+    cy.contains('Dictionary').click()
+    cy.contains('Categories').click()
+    cy.contains('CATEGORIES').should('exist')
+  })
+
+  it('4.1 - custom page', () => {
+    const site = `${Cypress.env('baseUrl')}${Cypress.env(
+      'DIALECT',
+    )}/custom/qacustompage`
+    cy.visit(`${Cypress.env('baseUrl')}`)
+    cy.contains('Sign in').click()
+    cy.wait(2000)
+    cy.login(
+      Cypress.env('CYPRESS_FV_USERNAME'),
+      Cypress.env('CYPRESS_FV_PASSWORD'),
+    )
+    cy.wait(3000)
+    cy.visit(`${Cypress.env('baseUrl')}${Cypress.env('DIALECT')}`)
+    cy.visit(site)
+    cy.contains('403').should('not.exist')
+    cy.contains('cf').click()
+    cy.contains('Sign out').click()
+    cy.visit(site)
+    cy.contains('403').should('exist')
+  })
+
+  it('5.1 - Check out word drawer', () => {
+    cy.visit(`${Cypress.env('baseUrl')}${Cypress.env('DIALECT')}`)
+
+    cy.contains('Dictionary').click()
+    cy.contains('Words').click()
+    cy.get('table tr td button', { timeout: 10000 }).first().click() // drawer should be open now
+
+    cy.get('[data-testid=DictionaryDetailPresentationDrawer]').should(
+      'be.visible',
+    )
+    cy.contains('MORE')
+    cy.contains('MORE').click()
+    cy.contains('SHARE')
+    cy.contains('SHARE').click()
+    cy.contains('Cancel').click()
+    cy.contains('MORE').click()
+    cy.contains('QR CODE')
+    cy.contains('QR CODE').click()
```

Comment:
Adding the second `.contains` for clicking is redundant in this block. 
Eg this: 
```
cy.contains('MORE')
cy.contains('MORE').click()
```
could be simplified to this:
```
cy.contains('MORE').click()
```